### PR TITLE
[webview-flutter] step 08: Removed used deprecated function

### PR DESCRIPTION
**Pull request by JDDV**
The use of `Scaffold.of(context).showSnackBar` is deprecated. This is changed to `ScaffoldMessenger.of(context).showSnackBar`.

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: webview_flutter/step_08/.gitignore [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
```
Comment (domesticmouse) [inline webview_flutter/step_08/.gitignore:1]: I'm curious why this is showing up as a delta. It's already landed in master?

**Review by domesticmouse — COMMENTED** [review]
This PR looks wrong, it's showing changed files that don't look like they've changed

**Comment by JDDV** [comment]
I just saw I made a mistake indeed, I'll make a new PR

**Comment by domesticmouse** [comment]
danke

**Pull request closed without merging**